packages/matchbox/src/components/Snackbar/Snackbar.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import { Close } from '@sparkpost/matchbox-icons';
+import { Close, Info, CheckCircle, Warning, Error as ErrorIcon } from '@sparkpost/matchbox-icons';
```

Comment:
`ErrorIcon` is now an exported alias  for `Error`, no need to rename this anymore

Reply:
OK, resolve :thumbsup: